src/BlueberryStaking.sol

```diff
@@ -632,21 +586,31 @@ contract BlueberryStaking is
             (10 ** (18 - stableDecimals));
     }
 
-    /*//////////////////////////////////////////////////
-                         MANAGEMENT
-    //////////////////////////////////////////////////*/
-
     /// @inheritdoc IBlueberryStaking
-    function changeBLB(address _blb) external onlyOwner {
-        if (_blb == address(0)) {
-            revert AddressZero();
-        }
-        blb = IBlueberryToken(_blb);
+    function getAccumulatedRewards(
+        address _user
+    ) external view returns (uint256 _totalRewards) {
+        address[] memory cachedTokens = ibTokens;
+        uint256 cachedLength = cachedTokens.length;
 
-        emit BLBUpdated(_blb, block.timestamp);
+        for (uint256 i; i < cachedLength; ++i) {
+            if (isIbToken[cachedTokens[i]]) {
+                _totalRewards += _earned(_user, cachedTokens[i]);
+            }
+        }
     }
 
-    /// @inheritdoc IBlueberryStaking
+    /*//////////////////////////////////////////////////
+                         MANAGEMENT
+    //////////////////////////////////////////////////*/
+
+    /**
+     * @notice Adds the given tokens to the list of ibTokens and sets the reward amounts for each token in the current
+     *        reward period
+     * @dev If the reward duration is over, you must call `modifyRewardAmount` after adding the token.
```

Comment:
What do you mean by this `@dev` comment? Adding new tokens will always kick off a new reward period for those tokens, so there is no need to follow this call with `modifyRewardAmount()`.

Reply:
This is old comments ported over from the interface before `addIbTokens` also modified reward amounts. Will remove.